faq/templates/faq/faq_detail.html

```diff
@@ -0,0 +1,15 @@
+{% extends "jmbo/modelbase_detail.html" %}
```

Comment:
This is the wrong path. You want `templates/faq/inclusion_tags/faq_detail.html`. Look at how jmbo-post does it.